userWhitelist.json

```diff
@@ -1,4 +1,14 @@
 {
+  "releaseCoordinators": [
+    "nithusha21",
+    "Showtim3",
+    "aks681",
+    "U8NWXD",
+    "vojtechjelinek",
+    "kevintab95",
+    "ankita240796",
+    "DubeySandeep"
```

Comment:
Update the list to have: nithusha21, aks681, vojtechjelinek, ankita240796, DubeySandeep, BenHenning (Others work as QA co-ordinators.)

Reply:
Done.